### PR TITLE
ATM90E32 advanced features

### PR DESCRIPTION
**Pull request by descipher**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#5670

## Checklist:

  - [ ] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [x] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/atm90e32.rst [hunk 1]
```diff
@@ -50,6 +50,12 @@ Configuration variables:
     :ref:`Sensor <config-sensor>`.
   - **power_factor** (*Optional*): Use the power factor value on this phase. All options from
     :ref:`Sensor <config-sensor>`.
+  - **phash_angle** (*Optional*): Use the phase angle value on this phase in degrees. All options from
```
Comment (jesserockz) [inline components/sensor/atm90e32.rst]:
```suggestion
  - **phase_angle** (*Optional*): Use the phase angle value on this phase in degrees. All options from
```

File: components/sensor/atm90e32.rst [hunk 2]
```diff
@@ -63,6 +69,8 @@ Configuration variables:
 - **phase_c** (*Optional*): The configuration options for the 3rd phase. Same options as 1st phase.
 - **frequency** (*Optional*): Use the frequenycy value calculated by the meter. All options from
   :ref:`Sensor <config-sensor>`.
+- **peak_current_signed** (*Optional*): Control the peak current output as signed or absolute. All options from
+  :ref:`Sensor <config-sensor>`.
```
Comment (jesserockz) [inline components/sensor/atm90e32.rst]: This doesnt exist in the code PR?

**Review by jesserockz — CHANGES_REQUESTED** [review]

**Comment by jesserockz** [comment]
Also, this PR is targeting the incorrect branch. It should be targeting `next`

Jesse

File: components/sensor/atm90e32.rst (hunk 2 above)
Comment (descipher) [reply components/sensor/atm90e32.rst]: Hmm, I will check what happened, it is on one of my local repos, I may have added it to the wrong local branch.

**Comment by descipher** [comment]
> Also, this PR is targeting the incorrect branch. It should be targeting `next`
> 
> Jesse

Would you like me to cancel it and re-submit to next? I believe you have the perms to change it but I do not see that option.

File: components/sensor/atm90e32.rst (hunk 2 above)
Comment (descipher) [reply components/sensor/atm90e32.rst]: Thanks for the eagle eyes on this. Ok I have cherry picked the missing piece from the local copy.

**Comment by jesserockz** [comment]
Yeah its in a weird place to change. Basically hit `edit` at the top, which changes the PR title to a text box, and the branch to a dropdown.

image

image

**Comment by jesserockz** [comment]
You might have to rebase locally

**Comment by descipher** [comment]
Thanks. Yes very weird spot. I was looking at that but it would not pull show the pull down for me. Fixing the conflicts shortly.

**Comment by descipher** [comment]
Somehow messed this up.  @jesserockz  can you please check if the PR is Ok. I needed to hard reset to a reflog point to fix it.. looks Ok to me.

File: components/sensor/atm90e32.rst (hunk 2 above)
Comment (jesserockz) [inline components/sensor/atm90e32.rst]:
```suggestion
- **peak_current_signed** (*Optional*, boolean): Control the peak current output as signed or absolute. Defaults to ``false``.
```

**Review by jesserockz — APPROVED** [review]

**Comment by descipher** [comment]
Thank-You @jesserockz - your high quality reviews are appreciated.

**Pull request merged**